src/test_rng.h

```diff
@@ -0,0 +1,22 @@
+#include <R.h>
+#include <Rinternals.h>
+
+#ifdef __cplusplus
```

Comment:
Also noexcept here I think. Can be neater with a single define at the top:
```
#ifdef __cplusplus
#define EXTERN
#define EXCEPT
#else
#define EXTERN extern "C"
#define EXCEPT noexcept
#endif
```
then use `EXTERN` and `EXCEPT` around each function

Reply:
Obsolete given #13